services/credentials-service/src/credentials/credentials.service.ts

```diff
@@ -319,4 +319,44 @@ export class CredentialsService {
       return { id: cred.id, ...signed };
     });
   }
+
+  async getRevocationList(
+    issuerId: string,
+    page = 1,
+    limit= 1000,
+  ){
+    let revocationList: any;
```

Comment:
Dont use Any. Use explicit Type. This is an array of strings right?

Reply:
Have made the changes its and array of Objects with the object having the details like DID, status and Issuer.